docs/discover/search.asciidoc

```diff
@@ -66,7 +79,6 @@ To load a saved search into Discover:
 If the saved search is associated with a different index pattern than is currently
 selected, opening the saved search also changes the selected index pattern.
```

Comment:
How does Kuery behave here? If there are saved searches that were saved with Lucene syntax - does Discover still use Lucene? It might be worth adding a note here "if you're using Kuery, blah blah"